packages/storybook/package.json

```diff
@@ -60,6 +60,11 @@
     "build-storybook:watch": "NODE_OPTIONS='--max-old-space-size=4096' build-storybook -w -s public --quiet",
     "test": "NODE_ENV=test jest"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/accordproject/web-components",
+    "directory": "packages/storybook"
+  },
```

Comment:
Similar to the dingus in `markdown-transform`, I don't think we want to bother with `storybook`

Reply:
So should I revert changes for this file?